README.md

```diff
@@ -1,57 +1,86 @@
 # libsir
 
+libsir &mdash; a cross-platform, thread-safe logging library
```

Comment:
I was taught that mdash does not have spaces on either side. I guess that's probably come down to a 'style guide' type issue.

Reply:
Easy fix